packages/labs/task/src/task.ts

```diff
@@ -34,16 +34,17 @@ export const initialState = Symbol();
 export type TaskStatus = typeof TaskStatus[keyof typeof TaskStatus];
 
 export type StatusRenderer<R> = {
-  initial?: () => unknown;
-  pending?: () => unknown;
-  complete?: (value: R) => unknown;
+  initial?: (value?: R) => unknown;
+  pending?: (value?: R) => unknown;
+  complete?: (value?: R) => unknown;
```

Comment:
I don't really like that the `complete` value is now optional.

Reply:
Is making `value` in `complete` optional related to allowing an initial value? If not, let's not change that.

Reply:
updated